app/views/system/schedulereport.html

```diff
@@ -0,0 +1,215 @@
+<div class="content-container" >
+        <link rel="stylesheet" href="https://formden.com/static/cdn/bootstrap-iso.css" /> 
+            <style>.bootstrap-iso .formden_header h2, .bootstrap-iso .formden_header p, .bootstrap-iso form{font-family: Arial, Helvetica, sans-serif; color: black}.bootstrap-iso form button, .bootstrap-iso form button:hover{color: white !important;} .asteriskField{color: red;}</style>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.6.5/angular.js"></script>
```

Comment:
Why these files are included here ?

Reply:
These is a form library cdn!

Reply:
Why we need another form library ? we are using Angular bootstrap here

Reply:
I am removing the Jquery and Angular CDN